target/linux/ath79/image/generic-tp-link.mk

```diff
@@ -111,6 +111,17 @@ define Device/tplink_archer-c6-v2-us
 endef
 TARGET_DEVICES += tplink_archer-c6-v2-us
 
+define Device/tplink_tl-wa1201-v2
+  $(Device/tplink-safeloader-uimage)
+  SOC := qca9563
+  IMAGE_SIZE := 7808k
```

Comment:
This does not match your partition table. Have you tested this device?

Reply:
I'm currently running it on two such devices, connecting two networks over 5ghz wds mode and regular access points on 2.4 ghz. They are up and running with like 2 days of uptime each.

Anyway I'll fix this entry.

Reply:
At least the ART partition should anyway never be modified by the user which is why it works anyway. I basically just virtually reduced the available image size.

Reply:
IMAGE_SIZE is just meant for checking whether the image file gets too big ("check-size"). In this context, it's mainly helpful for telling me that your setup is inconsistent.